src/plugins/home/public/application/components/sample_data_view_data_button.js

```diff
@@ -69,6 +69,9 @@ export class SampleDataViewDataButton extends React.Component {
         onClick: createAppNavigationHandler(path),
       };
     });
+
+    /** @typedef {import('@elastic/eui').EuiContextMenuProps['panels']} EuiContextMenuPanels */
+    /** @type {EuiContextMenuPanels} */
```

Comment:
Is this addition really necessary?

Reply:
We added just for type checking purposes